addons/skin.pm3.hd/720p/Home.xml

```diff
@@ -671,13 +671,15 @@
 						<thumb>$INFO[Skin.String(Home_Custom_Back_Programs_Folder)]</thumb>
 						<visible>!Skin.HasSetting(homepageHideProgramButton)</visible>
 					</item>
+<!--
 					<item id="5">
 						<label>8</label>
 						<onclick>ActivateWindow(Weather)</onclick>
 						<icon>special://skin/backgrounds/weather.jpg</icon>
 						<thumb>$INFO[Skin.String(Home_Custom_Back_Weather_Folder)]</thumb>
 						<visible>!Skin.HasSetting(homepageHideWeatherButton) + !IsEmpty(Weather.Plugin)</visible>
```

Comment:
once you squash this into one commit, I will still be complaining about this line :)